container/docs/conf.py

```diff
@@ -0,0 +1,310 @@
+# -*- coding: utf-8 -*-
```

Comment:
(No action required)

At some point we'll need to give this file more scrutiny (during the repo split) but for now it's fine.